ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -122,7 +135,24 @@
   [context task-id]
   (get-bulk-update-task-status (context->db context) task-id))
 
+(defn-timed get-bulk-update-collection-statuses-for-task
+  [context task-id]
+  (get-bulk-update-task-collection-status (context->db context) task-id))
+
 (defn-timed create-bulk-update-status
```

Comment:
How about rename this function to `create-bulk-update-task`? It is more about persisting the bulk update task in db than creating the statuses.